index.html

```diff
@@ -337,7 +347,7 @@
           scheduleObj.room = "";
         }
         //Add in photos if the user's schedule is being rendered
-        if (type == "user") {
+        if (type == "user" || type == "student") {
```

Comment:
How do student schedule views differ from teacher schedule views? Do you need to have 3 kinds?